packs/feat-effects/effect-crystal-luminescence.json

```
@@ -0,0 +1,49 @@
+{
+    "_id": "1bzFIIEw2K1CAUe9",
+    "img": "systems/pf2e/icons/spells/prismatic-wall.webp",
+    "name": "Effect: Crystal Luminescence",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Crystal Luminescence]</p>\n<p>If a spell or ability would activate your glowing horn while Crystal Luminescence is active, it instead increases the radius of the bright light and dim light by 10 feet each until the start of your next turn. This isn't cumulative, so using another such ability doesn't increase the radius again.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": ""
+        },
+        "rules": [
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.crystalLuminescence.brightRadius",
+                "value": 30
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.crystalLuminescence.dimRadius",
+                "value": 50
+            }
```

```suggestion
            {
                "key": "ActiveEffectLike",
                "mode": "add",
                "path": "flags.pf2e.crystalLuminescence.brightRadius",
                "value": 10
            },
            {
                "key": "ActiveEffectLike",
                "mode": "add",
                "path": "flags.pf2e.crystalLuminescence.dimRadius",
                "value": 10
            }
```
Override would cause headaches with taking the feat twice.